src/sentry/static/sentry/app/views/projectsDashboard/projectCard.tsx

```diff
@@ -145,15 +144,27 @@ type ContainerState = {
   projectDetails: Project | null;
 };
 
-const ProjectCardContainer = createReactClass<ContainerProps, ContainerState>({
-  mixins: [Reflux.listenTo(ProjectsStatsStore, 'onProjectStoreUpdate') as any],
+class ProjectCardContainer extends React.Component<ContainerProps, ContainerState> {
+  state = this.getInitialState();
+
   getInitialState(): ContainerState {
     const {project} = this.props;
     const initialState = ProjectsStatsStore.getInitialState() || {};
     return {
       projectDetails: initialState[project.slug] || null,
     };
-  },
+  }
+
+  componentWillUnmount() {
+    this.listeners.forEach(callIfFunction);
+  }
+
+  listeners = [
+    ProjectsStatsStore.listen(itemsBySlug => {
+      this.onProjectStoreUpdate(itemsBySlug);
+    }, undefined),
+  ];
+
   onProjectStoreUpdate(itemsBySlug: typeof ProjectsStatsStore['itemsBySlug']) {
```

Comment:
```suggestion
  onProjectStoreUpdate = (itemsBySlug: typeof ProjectsStatsStore['itemsBySlug']) => {
```

Reply:
just tried this style and it didn't like it for whatever reason